src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/Snapshots/TargetFrameworkInfo.cs

```diff
@@ -9,16 +9,17 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore
     ///     Represents the restore data for a single target framework in <see cref="UnconfiguredProject"/>.
     /// </summary>
     [DebuggerDisplay("TargetFrameworkMoniker = {TargetFrameworkMoniker}")]
-    internal class TargetFrameworkInfo : IVsTargetFrameworkInfo, IVsTargetFrameworkInfo2
+    internal class TargetFrameworkInfo : IVsTargetFrameworkInfo3
     {
         // If additional fields/properties are added to this class, please update RestoreHasher
-        public TargetFrameworkInfo(string targetFrameworkMoniker, IVsReferenceItems frameworkReferences, IVsReferenceItems packageDownloads, IVsReferenceItems projectReferences, IVsReferenceItems packageReferences, IVsProjectProperties properties)
+        public TargetFrameworkInfo(string targetFrameworkMoniker, IVsReferenceItems frameworkReferences, IVsReferenceItems packageDownloads, IVsReferenceItems projectReferences, IVsReferenceItems packageReferences, IVsReferenceItems centralPackageVersions, IVsProjectProperties properties)
         {
             TargetFrameworkMoniker = targetFrameworkMoniker;
             FrameworkReferences = frameworkReferences;
             PackageDownloads = packageDownloads;
             ProjectReferences = projectReferences;
             PackageReferences = packageReferences;
+            CentralPackageVersions = centralPackageVersions;
```

Comment:
Nit: Are these "Central"? They're just PackageVersion items, right? Can a project not use PackageVersion items to control the version of transient packages, regardless of whether they're defined centrally or not?

Reply:
Okay, I see the msbuild task has Central in the name, so i guess this is fine.. though maybe that should change  too :)

Reply:
This is an interface so I have to use this name, feedback to @cristinamanum.